src/types/election/published.ts

```diff
@@ -2,12 +2,27 @@ import { Election, IElectionParameters, IElectionType, IVoteType } from './elect
 import { MultiLanguage } from '../../util/lang';
 import { IQuestion } from '../metadata/election';
 import { PublishedCensus } from '../census/published';
-import { ElectionStatus } from '../../core/election';
+
+export enum ElectionStatus {
+  PROCESS_UNKNOWN = 'PROCESS_UNKNOWN',
+  UPCOMING = 'UPCOMING',
+  ONGOING = 'ONGOING',
+  ENDED = 'ENDED',
+  CANCELED = 'CANCELED',
+  PAUSED = 'PAUSED',
+  RESULTS = 'RESULTS',
+}
+
+export enum ElectionStatusReady {
```

Comment:
@marcvelmer can you clarifiy the point of logically separating `ElectionStatusReady` from `ElectionStatus` ?